packages/devtools/src/schema/parser.ts

```diff
@@ -482,6 +488,7 @@ export class SchemaParser {
     directives: Array<DirectiveAnnotation>,
   ): DefinitionWithReferences<ItemDefinition> {
     const required = isNonNullType(type)
+    const immutable = directives.some((item) => item.name === 'immutable')
```

Comment:
I think we probably need an additional check somewhere that the directive is only used on models (having the `@createModel` directive present), as we don't support it on nested objects.

Reply:
Please check my "feat:check if nested object" commit, I take the object directives to check for the `createModel` value.